src/jobs/heartbeat/heartbeat.ts

```diff
@@ -0,0 +1,15 @@
+import * as Sentry from '@sentry/node';
+
+import { bolt } from '@/api/slack';
+import { INFRA_HUB_HEARTBEAT_CHANNEL } from '@/config';
+
+export async function heartbeat() {
+  try {
+    await bolt.client.chat.postMessage({
+      channel: INFRA_HUB_HEARTBEAT_CHANNEL,
+      text: 'Infra Hub is up',
+    });
```

Comment:
What's the goal of sending this message? 
Is it about testing the Slack integration or are people supposed to notice these messages are not coming through ? I would push back against the second.